aries-site/src/layouts/main/Footer.js

```diff
@@ -1,12 +1,24 @@
 import React, { useContext } from 'react';
 import Link from 'next/link';
-import { NavLink } from 'aries-core';
+import { FooterLink } from 'aries-core';
 import { Box, Footer as GrommetFooter, ResponsiveContext, Text } from 'grommet';
 import { nameToPath } from '../../utils';
 
 export const Footer = () => {
   const size = useContext(ResponsiveContext);
   const year = new Date().getFullYear();
+
+  const footerLinks = [
```

Comment:
Should we rename this to be `externalFooterLinks` to be clear why `Feedback` isn't part of this?

Reply:
yes good suggestion Ill change that